sample/src/main/java/com/twilio/verify/sample/view/factors/create/CreateFactorFragment.kt

```diff
@@ -148,12 +147,15 @@ class CreateFactorFragment : Fragment() {
 
   private fun handleNetworkException(exception: TwilioVerifyException) {
     (exception.cause as? NetworkException)?.failureResponse?.apiError?.let {
-      Snackbar.make(
+      val snackbar = Snackbar.make(
         content,
         "Code: ${it.code} - ${it.message}",
-        BaseTransientBottomBar.LENGTH_LONG
+        BaseTransientBottomBar.LENGTH_INDEFINITE
```

Comment:
Should it be `Snackbar.LENGTH_INDEFINITE`?

Reply:
Changed